### DISCUSSION TITLE
I seemingly lost HEIC thumbnails once again, after upgrading AIO.

### DISCUSSION THREAD

**Discussion by Byter3** [nextcloud/all-in-one#3058] [Questions] [answered]
I seemingly lost HEIC thumbnails once again, after upgrading AIO.
I'm currently re-running the preview pregenerate command. It's gonna take a day at least. I'm gonna update this post after it's done.

**Comment by szaimen** [comment, marked as answer]
Reported in https://github.com/nextcloud/all-in-one/issues/3060
